.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
\ No newline at end of file
+  - "node"
+  - "iojs"
```

Comment:
It's better to be explicit about which versions we're testing on, so please remove `node` and `iojs` from the list, and add `7`. That way people can see at which versions we know it works on. It's likely that there will be a few more major node versions before there is another change to asciify, so it's clearer if we state the last explicit version it was tested on, even if we assume it'll work on all future versions.